### PR TITLE
Custom layout update changes

### PR DESCRIPTION
**Pull request by ccondra**
The purpose of this PR is to update the Layout Update screenshots and instructions to reflect the UI and process changes.  I removed all code examples and now refer to the Custom Layout topics in the Frontend Developers Guide for more information. 

This PR is related to:

- [Custom layout update changes](https://github.com/magento/merchdocs/issues/314)  (User Guide)
- [Custom Layout Edit Updates Deprecated ](https://github.com/magento/merchdocs/issues/314) (DevDocs)

Updated topics:

1. [Layout Updates](https://docs.magento.com/m2/ee/user_guide/design/layout-updates.html)
- Added link to [Layout overview](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/layout-overview.html) in devdocs.
- Removed out of date info and added brief description of the new process.
- Added content from Standard Block Layout

2. [Adding a New Page](https://docs.magento.com/m2/ee/user_guide/cms/page-add.html)
- Updated screenshot - page-design.png
- Updated instructions for new Custom Layout Update dropdown and New Theme field.
- Corrected conditionals for ce/ee/b2b that weren't working correctly.

3. [Using a Layout Update](https://docs.magento.com/m2/ee/user_guide/design/layout-update-place-block.html)
- Reworked topic for new process with links to devdocs.

4. [Blocks](https://docs.magento.com/m2/ee/user_guide/cms/block-add.html)
- Added link to Using a Layout Update.

5. [Design Settings](https://docs.magento.com/m2/ee/user_guide/catalog/categories-custom-design.html)  (for category)
- Updated screenshot category-design.png
- Updated field descriptions - removed Layout Update XML field and added Custom Layout Options.

6. [Design Settings](https://docs.magento.com/m2/ee/user_guide/catalog/settings-advanced-design.html) (for product)
- Updated screenshot category-design.png
- Updated field descriptions - removed Layout Update XML field and added Custom Layout Options.

7. Redirected and removed from TOC:
     This content was moved to the Layout Updates topic:
  - /design/block-layout-standard.html

     The following topics are obsolete and now addressed in the Frontend Developers Guide:
  - /design/layout-update-examples.html
  - /design/layout-update-syntax.html
  - /design/layout-update-block-order.html
  - /design/xml-load-sequence.html

8. Updated [Change Log](https://docs.magento.com/m2/ee/user_guide/magento/change-log.html)

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

## Additional information
Staging: #423

### CODE REVIEW

File: src/design/layout-updates.md [hunk 1]
```diff
@@ -1,10 +1,37 @@
...
+The [Widget]({% link cms/widgets.md %}) tool is an easy way to add an existing [content block]({% link cms/blocks.md %}) to the default layout of a page. For more advanced updates, you must save the XML layout update code on the server, and then reference the file as a custom layout update from the Admin. For an overview of the process, see [Making Layout Updates]({% link design/layout-update-place-block.md %}).
 
-In many cases, the same result can be achieved with the [Widget]({% link cms/widgets.md %}) tool. To place a block of content as a widget, you must identify the page, and the location on the page where you want the block to appear. You can use the Widget tool to place a block on most any page in your store, including the home page and all content pages. However, to place a block in the sidebar of a specific page, you must make the change by entering code as a layout update.
+In the following diagram, the names that refer to contiainers are black, and the block types, or block class paths, are blue.
```
Comment (leslietilling) [inline src/design/layout-updates.md]:
```suggestion
In the following diagram, the names that refer to containers are black and the block types, or block class paths, are blue.
```

File: src/design/layout-updates.md [hunk 2]
```diff
@@ -1,10 +1,37 @@
...
+|page/template_links|There are two blocks of this type in the standard layout. The `top.links` block is a child of the header block, and corresponds to the top navigation menu. The `footer_links` block is a child of the footer block, and corresponds to the bottom navigation menu. <br/><br/>**_Note:_** It is possible to manipulate the template links, as shown in the examples.|
+|page/switch|There are two blocks of this type in a standard layout. The `store_language` block is a child of the header block, and corresponds to the top language switcher. The `store_switcher` block is a child of the footer block, and corresponds to the bottom store switcher.|
+|core/messages|There are two blocks of this type in a standard layout. The `global_messages` block displays global messages. The `messages` block is used to display all other messages. If you remove these blocks, the customer won't be able to see any messages.|
+|core/text_list|This type of block is widely used throughout Magento, and is used as a placeholder for rendering children blocks.|
```
Comment (leslietilling) [inline src/design/layout-updates.md]:
```suggestion
|core/text_list|This type of block is widely used throughout Magento as a placeholder for rendering children blocks.|
```

File: src/design/layout-update-place-block.md [hunk 3]
```diff
@@ -1,41 +1,55 @@
...
-1. In the grid, find the block you want to place, and take note of its **Identifier**. Make sure that you have the correct spelling.
+## Step 2: Compose the layout update in XML
+
+1. Compose the layout instructions in XML to [Reference a CMS Block][3].
+
+1. Save the [layout instructions][2] on the server in the layout folder where XML files are saved for the theme. For example:
+
+   `<theme_dir>/<Namespace>_<Module>/layout`
+
+   The [layout handle][4] is the filename that begins with the `cms_page_view_selectable_`, followed by the URL key of the CMS page, the layout update option, andrake the  `xml` file suffix. In the following example, `customer-service` is the URL key of the page, and `ChatTool` is the option that you select to apply the layout update to the page.
```
Comment (leslietilling) [inline src/design/layout-update-place-block.md]: not sure about `andrake`, should this be `and rake`?

File: src/cms/blocks.md [hunk 4]
```diff
@@ -4,7 +4,9 @@ title: Blocks
 
 A block is a modular unit of content that can be positioned most anywhere on the page. Content blocks are sometimes referred to as static blocks, or CMS blocks. They can be used to display fixed information such as text, images, and embedded video, as well as dynamic information from a widget or that originates in a database or other source. Most elements on the home page are blocks that can be easily managed.
 
-You can create custom blocks of content without writing any code, and assign them to appear in a specific place in the page layout. Blocks can also be defined and positioned by making a [layout update]({% link design/layout-updates.md %}) in XML code.
+You can create custom blocks of content without writing any code, and assign them to appear in a specific place in the page layout. Blocks can be positioned with a [Widget]({% link cms/widget-static-block.md %}) or by composing a [layout update]({% link design/layout-update-place-block.md %}) in XML and saving it on the server.  To learn more, see the [Layout][1] section in the developer documentation.
```
Comment (leslietilling) [inline src/cms/blocks.md]: Do you mean inside a widget?

File: src/catalog/categories-custom-design.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ _Scheduled Design Update_
...
-|Layout Update XML|Updates the theme [layout with custom XML]({% link design/layout-updates.md %}) code.|
+|Custom Layout Update|Lists the avaiable custom layout update files on the server. Choose the custom layout update that you want to apply to the category.|
```
Comment (leslietilling) [inline src/catalog/categories-custom-design.md]:
```suggestion
|Custom Layout Update|Lists the available custom layout update files on the server. Choose the custom layout update that you want to apply to the category.|
```

File: src/design/layout-update-place-block.md (hunk 3 above)
Comment (ccondra) [reply src/design/layout-update-place-block.md]: looks like I was going to run rake in the terminal, but my cursor was in the wrong place!

Comment (ccondra) [reply src/design/layout-update-place-block.md]: fixed.

File: src/cms/blocks.md (hunk 4 above)
Comment (ccondra) [reply src/cms/blocks.md]: No, "with" a widget is accurate.  I could change it to "with the Widget tool..."

Comment (ccondra) [reply src/cms/blocks.md]: Fixed.

File: src/magento/change-log.md [hunk 6]
```diff
@@ -4,6 +4,16 @@ title: Change Log
 
 Our documentation is continually updated with new topics, clarifications, and corrections to existing content. Learn more about new features, major updates, and releases, organized by month and year. Check back every now and then to see what’s new.
 
+## March 2020
+
+### Updated topics
+
+|Topic |Change|
+|--- |--- |
+|[Layout Updates]({% link design/layout-updates.md %})|Updated all topics in this section to reflect the change in layout update implementation. Layout updates are now saved on the server and selected from a list. For help composing layout update code, see the [Layout](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/layout-overview.html) section in the Frontend Developer Guide.|
```
Comment (leslietilling) [inline src/magento/change-log.md]: Let's leave out the sentence about DevDocs. Just a short description of what is updated here.

Comment (ccondra) [reply src/magento/change-log.md]: Fixed.

**Comment by dshevtsov** [comment]
Sent a request to GitHub support regarding the failed and unexpected duplicate check: _Ticket ID: 614698_

**Comment by ccondra** [comment]
running tests

**Pull request merged**